Makefile

```diff
@@ -23,33 +29,38 @@ console:
 db:
 	@$(BASH_DEV) "bundle exec rake db:setup db:migrate"
 
+.PHONY: down
+down:
+	@$(COMPOSE_DEV) down
+
 .PHONY: guard
-guard: db
+guard:
 	@$(BASH_DEV) "bundle exec guard"
 
 .PHONY: lint
-lint: db
+lint:
 	@$(BASH_DEV) "bundle exec rake lint"
 
+.PHONY: migrate
+migrate:
+	@$(BASH_TEST) "bundle exec rake db:migrate"
+
+.PHONY: rebuild
+rebuild: down
+	@$(COMPOSE_DEV) build
+
 .PHONY: security
-security: db
+security:
 	@$(BASH_DEV) "bundle exec rake security"
 
+.PHONY: server
+server:
+	@$(BASH_DEV) "rm -f tmp/pids/server.pid && bundle exec rails server"
```

Comment:
So the difference between `server` and `up` is that server just runs the API server and the other runs both API and Sidekiq? Or is there some other benefit here?

I'm fairly confident that the pidfile problem is from foreman (which I think is unnecessary in the first place but we haven't replaced it yet), so I don't _think_ the rm is necessary.

Reply:
I thought so too at first, but it happens when running the rails server way too 😐  https://github.com/docker/compose/issues/1393

Reply:
Bummer.